test/test.tf

```diff
@@ -0,0 +1,72 @@
+variable "dns_zone" {
+  default = "Z2J3KVPABDNIL1"
+}
+
+provider "aws" {
+  version                 = "1.55"
+  region                  = "ap-southeast-2"
+  alias                   = "ap-southeast-2"
+  shared_credentials_file = "/aws/credentials"
+  profile                 = "aeternity"
+}
+
+variable "vault_addr" {
+  description = "Vault server URL address"
+}
+
+variable "bootstrap_version" {
+  default = "v2.0.1"
+}
+
+variable "test_gateway_dns" {
+  default = "api.test.ops.aeternity.com"
```

Comment:
I don't think this can have default, 2 environments should be able to co-exists, so similar to envid, this has to be unique as well

Reply:
changed in 4a2d123